PWGCF/Flow/Tasks/FlowTask.cxx

```diff
@@ -559,18 +565,22 @@ struct FlowTask {
   template <typename TTrack>
   bool trackSelected(TTrack track)
   {
-    if (cfgFilterFlag == 0)
-      return track.isGlobalTrack();
-    else if (cfgFilterFlag == 1)
-      return (track.isGlobalTrackSDD() == (uint8_t) true);
-    else if (cfgFilterFlag == 2)
-      return (track.isGlobalTrackWoTPCCluster() && track.tpcNClsFound() >= cfgCutTPCclu);
-    else if (cfgFilterFlag == 3)
-      return (track.isGlobalTrackWoPtEta() && (abs(track.eta()) < cfgCutEta) && (track.pt() > cfgCutPtMin) && (track.pt() < cfgCutPtMax));
-    else if (cfgFilterFlag == 4)
-      return (track.isGlobalTrackWoDCA() && track.dcaZ() <= cfgCutDCAz && track.dcaXY() <= cfgCutDCAxy * pow(track.pt(), -1.1));
-    else if (cfgFilterFlag == 5)
-      return (track.isGlobalTrackWoDCATPCCluster() && track.dcaZ() <= cfgCutDCAz && track.dcaXY() <= cfgCutDCAxy * pow(track.pt(), -1.1) && track.tpcNClsFound() >= cfgCutTPCclu);
+    switch (cfgFilterFlag) {
+      case 0:
+        return track.isGlobalTrack();
+      case 1:
+        return (track.isGlobalTrackSDD() == (uint8_t) true);
+      case 2:
+        return (track.isGlobalTrackWoTPCCluster() && track.tpcNClsFound() >= cfgCutTPCclu);
+      case 3:
+        return (track.isGlobalTrackWoPtEta() && (abs(track.eta()) < cfgCutEta) && (track.pt() > cfgCutPtMin) && (track.pt() < cfgCutPtMax));
+      case 4:
+        return (track.isGlobalTrackWoDCA() && abs(track.dcaZ()) <= cfgCutDCAz && abs(track.dcaXY()) <= cfgCutDCAxy * pow(track.pt(), -1.1));
+      case 5:
+        return (track.isGlobalTrackWoDCATPCCluster() && abs(track.dcaZ()) <= cfgCutDCAz && abs(track.dcaXY()) <= cfgCutDCAxy * pow(track.pt(), -1.1) && track.tpcNClsFound() >= cfgCutTPCclu);
+      default:
+        return false;
```

Comment:
Please, don't use `abs`. It turned out that now it is behaving as it should have, i.e. return an `int`
Use instead `std::fabs`